packages/beeq/src/components/date-picker/bq-date-picker.tsx

```diff
@@ -80,6 +81,8 @@ export class BqDatePicker {
   @State() hasPrefix = false;
   @State() hasSuffix = false;
   @State() hasValue = false;
+  @State() hasRangeEnd = false;
+  @State() setTentative = null;
```

Comment:
We should support passing an external value via prop/attribute since Cally supports that, meaning that instead  of adding a state, we could provide a mutable property:

```ts
/** The date that is tentatively selected e.g. the start of a range selection  */
  @Prop({ reflect: true, mutable: true }) tentative?: string;
```

Reply:
Don't forget to update Storybook with the new attribute.